app/grants/templates/grants/shared/landing_qf_active.html

```diff
@@ -0,0 +1,97 @@
+{% load humanize static i18n %}
+<div class="container bg-white">
+  <!--  Header section -->
+  <div class="my-5">
+    <div class="row justify-content-between align-items-center">
+      <div class="col">
+        <h1>
+          Democratic public goods funding <span class="text-nowrap">for the 21st century</span>
+        </h1>
+        <h3 class="text-wrap text-muted">
+          Gitcoin grants sustain web3 projects with
+          <span class="text-nowrap">
+            <a href="https://vitalik.ca/general/2019/12/07/quadratic.html">quadratic funding</a>
+          </span>
+        </h3>
+        <div class="my-3">
+          <a class="btn btn-primary" href="/grants/explore">{% trans 'Explore Grants' %}</a>
+          <a class="btn btn-outline-primary ml-3" href="/grants/new">{% trans 'Create a Grant' %}</a>
+        </div>
+        <div>
+          or
+          <a href="/grants/matching-partners/new">
+            {% trans "become a member of the funder's league" %}
+          </a>
+        </div>
+      </div>
+      <div class="col-auto d-none d-md-block">
+        <img src="{% static 'v2/images/grants/rocket.svg' %}" />
+      </div>
+    </div>
+  </div>
+
+  <!-- This div is where the hero image goes-->
+  <div class="pb-5" id="grant-hero-img">
+    <h1>$500k Funding Pool, Now Live</h1>
+    <div class="text-muted">Sponsored by</div>
+    <div class="my-5">
+      <div class="row">
+        <div class="col-12 col-md-6 order-md-last mb-5 mb-md-0">&nbsp;</div>
+      </div>
+      <div class="m-0 col-12 col-md-6 offset-lg-1 col-lg-5 d-none d-sm-block">&nbsp;</div>
+    </div>
+  </div>
+
+  <!-- Summary of matching pools this round -->
+  <div class="row justify-content-between align-items-center text-secondary mt-3">
+    <div class="col-auto">$125K Community</div>
+    <div class="col-auto">$125K Infra Tech</div>
+    <div class="col-auto">$125K dApp Tech</div>
+    <div class="col-auto">$125K Eask Asia</div>
+    <div class="col-auto">$125K NFTs</div>
+  </div>
```

Comment:
^ This shouldn't be hardcoded right ?

Reply:
ohh true this need to came from the filters, the info already exist on the api endpoint, good catch

Reply:
@mds1 I am getting this in for now => but we def want this from the API endpoint